dashboard/Makefile

```diff
@@ -19,18 +19,33 @@ DETACHED_MODE := $(if $(DETACHED),-d,)
 NODE_VERSION :=`cat ../.node-version`
 PWD := $(shell pwd)
 
+TEMPLATE_LOCATION := ./application/lib/ood_core/batch_connect/templates
+TURBOVNC_TEMPLATE := turbovnc.rb
+KVM_TEMPLATE      := kvm.rb
+
 remote-dev clean-remote-dev:     LOGIN_HOST   := login.rc.fas.harvard.edu
 remote-dev clean-remote-dev:     APP_FOLDER   := ./fasrc/dev/dashboard
+remote-dev:                      TEMPLATE_URL := https://vdi.rc.fas.harvard.edu/pun/sys/dashboard/files/fs/opt/ood/ondemand/root/usr/share/gems/2.7/ondemand/2.0.12/gems/ood_core-0.17.1/lib/ood_core/batch_connect/templates
```

Comment:
I cannot think any other way. I have added a check to make sure there is a valid template file coming back from the server. But with this approach we will need to update the Makefile on every OOD update.

As the actual templates will change less often, not sure it will be less troublesome to change the approach and have a copy of the files in the code base and copy the right version with Make.

Reply:
I like that you added a validation check, but I'd like to think about this a little more. How can we avoid introducing this troublesome and potentially unnecessary effort of multiple password entries and updating URL paths?

The problem we are trying to solve is to keep the `turbovnc.rb` and `kvm.rb` files from the default dashboard app in sync with the same files in the Sid dashboard app. Is there another opportunity for us to do so, other than at deployment time?

What about at runtime? Would it be possible for the Sid dashboard app itself to check the local filesystem to determine if the files differ, and raise a warning if so? This implementation would also address #[658](https://github.com/hmdc/DevOpsProjects/issues/658).

Reply:
We decided that if we do #658 first (to change the puppet manifest so that it deploys a copy of the `turbovnc.rb` and `kvm.rb` files directly in to the Sid app installation dir), then we will be able to access these files at a fixed path from the Cannon Staging and FASSE Staging servers. This will make this implementation of deploying the files to remote dev at build time more resilient.